assets/src/blocks/ActionCustomButtonText/edit.js

```diff
@@ -0,0 +1,15 @@
+import {useEntityProp} from '@wordpress/core-data';
+
+const {__} = wp.i18n;
+
+const DEFAULT_TEXT = window.p4_vars.options.take_action_covers_button_text || __('Take action', 'planet4-blocks');
+
+export default function Edit({context: {postId}}) {
+  const [metaFields] = useEntityProp('postType', 'p4_action', 'meta', postId);
+
+  return (
+    <button className="btn btn-primary btn-small">
```

Comment:
Since the whole card is clickable now, I haven't made this a link with the href, but now I'm thinking it would probably be better in terms of accessibility 🤔